js/data.js

```diff
@@ -886,14 +1017,14 @@ var data = {
         semester: [ss15],
         cp: 6,
         dozent: ["Dr. Matthias Trapp", "Amir Semmo", "Sebastian Pasewaldt"],
-        nameLV: "Interactive Representations of Data Structures and Algorithms"
+        nameLV: "Introduction to Visualization"
```

Comment:
i was also confused by the old name, i've never heard it before. indeed the official name seems to be "introduction to visualization", see https://hpi.de/studium/lehrveranstaltungen/it-systems-engineering/lehrveranstaltung/course/2015/sommersemester-2015-introduction-to-visualization.html